shared/tracker2/bio/index.js

```diff
@@ -12,13 +12,27 @@ type Props = {|
   fullname: ?string,
   inTracker: boolean,
   location: ?string,
+  registeredForAirdrop: ?boolean,
 |}
 
 const Bio = (p: Props) => (
   <Kb.Box2 direction="vertical" fullWidth={true} style={styles.container} centerChildren={true} gap="xtiny">
-    <Kb.Text type="BodyBig" lineClamp={p.inTracker ? 1 : undefined} style={styles.text} selectable={true}>
-      {p.fullname}
-    </Kb.Text>
+    <Kb.Box2 direction="horizontal" style={styles.fullNameContainer} gap="tiny">
+      <Kb.Text
+        center={true}
+        type="BodyBig"
+        lineClamp={p.inTracker ? 1 : undefined}
+        style={styles.text}
+        selectable={true}
+      >
+        {p.fullname}
+      </Kb.Text>
+      {p.registeredForAirdrop && (
```

Comment:
Disable if `!flags.airdrop`, also seems like the `centered` -> `center` change messed up the alignment.
<img width="212" alt="image" src="https://user-images.githubusercontent.com/11968340/53203720-ef548a00-35f7-11e9-8e78-974b5e06b962.png">